### PR TITLE
Calculator Restructured

### PR DESCRIPTION
**Pull request by Jagadheesh28**
Reduce calculator width and font color change

### CODE REVIEW

File: src/app/page.tsx [hunk 1]
```diff
@@ -43,27 +43,36 @@ export default function Home() {
...
-          readOnly
-        />
-        <input
-          type="text"
-          className="w-100 fs-3 fw-bold mb-4 text-danger"
-          value={result}
-          readOnly
-        />
-        <div className="grid grid-cols-4 gap-2">
+      <div className=".bg-white p-6 w-50">
```
Comment (Rino-makoitlab) [inline src/app/page.tsx:48]: Remove the dot in class

File: src/app/page.tsx [hunk 2]
```diff
@@ -43,27 +43,36 @@ export default function Home() {
     "=",
   ];
   return (
-    <main className="d-flex min-vh-100 flex-column align-items-center p-24 bg-primary">
+    <main className="d-flex min-vh-100 flex-column align-items-center p-24 bg-primary ">
```
Comment (Rino-makoitlab) [inline src/app/page.tsx]: Remove the space in last

File: src/app/page.tsx (hunk 1 above)
Comment (Rino-makoitlab) [inline src/app/page.tsx]: Remove the dot in class

**Review by Rino-makoitlab — COMMENTED** [review]
Resolve the items

File: src/app/page.tsx (hunk 2 above)
Comment (Jagadheesh28) [reply src/app/page.tsx]: I have removed the space in last

File: src/app/page.tsx (hunk 1 above)
Comment (Jagadheesh28) [reply src/app/page.tsx]: I have removed the Dot in class

**Pull request merged**